src/Symfony/Component/Validator/Constraints/Expression.php

```diff
@@ -54,12 +54,4 @@ public function getTargets()
     {
         return array(self::CLASS_CONSTRAINT, self::PROPERTY_CONSTRAINT);
     }
-
-    /**
-     * {@inheritdoc}
-     */
-    public function validatedBy()
-    {
-        return 'validator.expression';
```

Comment:
``symfony/dependency-injection`` does not do anything with aliases. It does not have any knowledge of it.

and no, you cannot break BC and add a note in the changelog. BC breaks are **forbidden** in minor versions (and we try to avoid hard BC breaks in 3.0, i.e. we only breaks BC by removing things already deprecated in 2.x before)